### PR TITLE
[SDK-1957] Update aspnet core quickstart to match sample

### PR DESCRIPTION
**Pull request by frederikprijck**
This PR updates the Quickstart to be aligned with the changes made to te sample project in https://github.com/auth0-samples/auth0-aspnetcore-mvc-samples/pull/60

- Use `ConfigureSameSiteNoneCookies` which is needed to make the samples work with Http
- Added a few rewordings
- Removed some code from snippets when it is not relevant and was already part of an earlier snippet
- Changed the way roles are configured to include assigning roles through the UI and use a rule to add those roles to the idToken
-

### CODE REVIEW

File: articles/quickstart/webapp/aspnet-core-3/03-authorization.md [hunk 1]
```diff
@@ -18,57 +18,25 @@ ASP.NET Core supports [Role based Authorization](https://docs.microsoft.com/en-u
...
 :::
 
-## Create a Rule to Assign Roles
+## Create and assign roles
 
-Create a rule that assigns the following access roles to your user:
-* An admin role
-* A regular user role
+Before we can add Role Based Access Control, we will need to ensure the required roles are created and assigned to the corresponding user(s).
+Follow the guidance explained in [assign-roles-to-users](/users/assign-roles-to-users) to ensure your user gets assigned the `admin` role.
```
Comment (stevehobbsdev) [inline articles/quickstart/webapp/aspnet-core-3/03-authorization.md]: I would remove the person out of the context here and be more impertive. "Before RBAC can be added, ensure the required roles are created.."

File: articles/quickstart/webapp/aspnet-core-3/_includes/_login.md [hunk 2]
```diff
@@ -219,7 +154,7 @@ public class AccountController : Controller
 
 ASP.NET Core calls `SignOutAsync` for the "Auth0" authentication scheme. You need to provide the OIDC middleware with the URL for logging the user out of Auth0. To set the URL, handle the `OnRedirectToIdentityProviderForSignOut` event when you register the OIDC authentication handler.
 
-When the application calls `SignOutAsync` for the OIDC middleware, it also calls the `/v2/logout` endpoint of the Auth0 Authentication API. The user is logged out of Auth0.
+When the application calls `SignOutAsync` for the OIDC middleware, it also calls the `/v2/logout` endpoint of the Auth0 Authentication API which will ensure the user is logged out of Auth0.
```
Comment (stevehobbsdev) [inline articles/quickstart/webapp/aspnet-core-3/_includes/_login.md]:
```suggestion
When the application calls `SignOutAsync` for the OIDC middleware, it also calls the `/v2/logout` endpoint of the Auth0 Authentication API, which will ensure the user is logged out of Auth0.
```

File: articles/quickstart/webapp/aspnet-core-3/_includes/_login.md [hunk 3]
```diff
@@ -328,13 +243,55 @@ When the user selects the **Log In** button, the OIDC middleware redirects them
...
+    })
+    .AddCookie()
+    .AddOpenIdConnect("Auth0", options => {
+        // ...
+
+        options.Events = new OpenIdConnectEvents
+        {
+            OnRedirectToIdentityProvider = context =>
+            {
+                context.ProtocolMessage.SetParameter("audience", Configuration["Auth0:Audience"]);
```
Comment (stevehobbsdev) [inline articles/quickstart/webapp/aspnet-core-3/_includes/_login.md:275]: Can we have a code comment here about why this line is needed? It's the crux of the snippet.

**Review by stevehobbsdev — CHANGES_REQUESTED** [review]
Just a few minor things. Mainly be consistent about heading casing and careful of mixture of we/you (I would write this out so you don't even have to deal with it).

File: articles/quickstart/webapp/aspnet-core-3/03-authorization.md (hunk 1 above)
Comment (frederikprijck) [reply articles/quickstart/webapp/aspnet-core-3/03-authorization.md]: As mentioned on Slack, I think this is a great addition but the majority of the quickstart is already written using 'you/your' so we have decided to just ensure the added/changed text is consistent and also used 'you/your'

File: articles/quickstart/webapp/aspnet-core-3/_includes/_login.md [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
+## Configure Your Application to Use Auth0
 
 [Universal Login](/hosted-pages/login) is the easiest way to set up authentication in your application. We recommend using it for the best experience, best security and the fullest array of features. This guide will use it to provide a way for your users to log in to your ASP.NET Core application.
 
 ::: note
 You can also create a custom login for prompting the user for their username and password. To learn how to do this in your application, follow the [Custom Login sample](https://github.com/auth0-samples/auth0-aspnetcore-mvc-samples/tree/master/Samples/custom-login).
 :::
 
-### Install dependencies
+### Install Dependencies
```
Comment (stevehobbsdev) [inline articles/quickstart/webapp/aspnet-core-3/_includes/_login.md]: From our discussion yesterday, only the level 2 headings need to be title case - level 3s and below should be sentence case. So this one and the couple others below can be reverted.

**Review by stevehobbsdev — CHANGES_REQUESTED** [review]
Looks good, just a minor thing about headers.

File: articles/quickstart/webapp/aspnet-core-3/_includes/_login.md [hunk 5]
```diff
@@ -140,44 +110,9 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
...
-        };
-    });
-}
-```
-
-## Trigger authentication
+## Trigger Authentication
 
-### Add the `Login` and `Logout` Methods
+### Add the Login and Logout methods
```
Comment (frederikprijck) [inline articles/quickstart/webapp/aspnet-core-3/_includes/_login.md:115]: I dropped the backticks here to not have backticks on headers.

**Review by stevehobbsdev — APPROVED** [review]

**Pull request merged**